src/config.rs

```diff
@@ -405,22 +410,31 @@ mod tests {
 
         assert!(config.get::<FullCommandArgs>("empty_array").is_err());
 
-        let args: FullCommandArgs = config.get("empty_string").unwrap();
-        assert_eq!(args, FullCommandArgs::String("".to_owned()));
-        assert_eq!(args.args(), [""].as_ref());
+        let command_args: FullCommandArgs = config.get("empty_string").unwrap();
```

Comment:
(Nit, not necessarily for this PR): Perhaps an empty string should return an error, like an empty array?

Reply:
It could check `""`, `[""]`, etc., but doesn't have to. `Command::new().spawn()` can deal with that.